Mathlib/AlgebraicTopology/SimplexCategory.lean

```diff
@@ -564,6 +564,50 @@ lemma δ_one_mkOfSucc {n : ℕ} (i : Fin n) :
   fin_cases x
   aesop
 
+/-- If `j > i + 1`, `mkOfSucc i ≫ δ j` is the morphism `[1] ⟶ [n]` that
+picks out arrow `i`. -/
+lemma mkOfSucc_δ_lt {n : ℕ} {i : Fin n} {j : Fin (n + 2)}
+    (h : i.succ.castSucc < j) :
+    mkOfSucc i ≫ δ j = mkOfSucc i.castSucc := by
+  ext x
+  fin_cases x
+  · simp [δ, Fin.succAbove_of_castSucc_lt _ _ (Nat.lt_trans _ h)]
+  · simp [δ, Fin.succAbove_of_castSucc_lt _ _ h]
+
+/-- If `j < i + 1`, `mkOfSucc i ≫ δ j` is the morphism `[1] ⟶ [n]` that
+picks out arrow `i + 1`. -/
+lemma mkOfSucc_δ_gt {n : ℕ} {i : Fin n} {j : Fin (n + 2)}
+    (h : j < i.succ.castSucc) :
+    mkOfSucc i ≫ δ j = mkOfSucc i.succ := by
+  ext x
+  simp only [δ, len_mk, mkHom, comp_toOrderHom, Hom.toOrderHom_mk, OrderHom.comp_coe,
+    OrderEmbedding.toOrderHom_coe, Function.comp_apply, Fin.succAboveOrderEmb_apply]
+  fin_cases x <;> rw [Fin.succAbove_of_le_castSucc]
+  · rfl
+  · exact Nat.le_of_lt_succ h
+  · rfl
+  · exact Nat.le_of_lt h
+
+/-- If `j = i + 1`, `mkOfSucc i ≫ δ j` is the morphism `[1] ⟶ [n]` that
+picks out the composite of arrows `i` and `i + 1`. -/
```

Comment:
```suggestion
which sends `0` and `1` to `i` and `i + 2` respectively. -/
```

Reply:
Would you recommend keeping the "picks out" language for the comments on the two lemmas above?

Reply:
I think the phrasing should also be changed because it should be phrased in the language of ordered sets.